app/Models/ProductInfo.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class ProductInfo extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'product_id',
+        'description',
+        'image_location',
+    ];
+
+    public function product(){
+        return $this->hasOne(Product::class);
```

Comment:
please take a look at the inverse of the relation
https://laravel.com/docs/8.x/eloquent-relationships#one-to-one-defining-the-inverse-of-the-relationship